utils/types/models.ts

```diff
@@ -54,9 +56,18 @@ export interface NumberAttribute extends Attribute {
   possibleValues: 'number'
 }
 
+export interface Log {
+  _id?: string
+  staff: string | UserResponse
```

Comment:
in the base types use `string | User` because we then override with `UserResponse` in the `LogResponse`

Reply:
True! Fixed.